scripts/pkg_in_pipe/.gitignore

```diff
@@ -0,0 +1 @@
+report.html
```

Comment:
Not needed anymore as the output is flexible, and the example in README.md is outside of the repo.

Reply:
while not strictly needed, I think it's still nice to have it for local development

Reply:
Yes, it is still the default output file.